core/app/services/spree/products/prepare_nested_attributes.rb

```
@@ -130,7 +130,7 @@ def update_option_value_variants(option_value_params, existing_variant)
         option_value_variant_params = {}
 
         option_value_params.each_with_index do |opt, index|
-          option_type = Spree::OptionType.where(name: opt[:name].parameterize).first_or_initialize do |o|
+          option_type = Spree::OptionType.find_by(id: opt.require(:id)) || Spree::OptionType.where(name: opt[:name].parameterize).first_or_initialize do |o|
```

RuboCop (1.73)

[convention] Line is too long. [156/150]

(Layout/LineLength)